server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -16,6 +16,7 @@
 // under the License.
 package com.cloud.vm;
 
+import static com.cloud.configuration.ConfigurationManagerImpl.VM_USERDATA_MAX_LENGTH;
```

Comment:
these import are not needed

Reply:
could be something during rebase, removing